tenable/io/v3/was/plugins/api.py

```diff
@@ -0,0 +1,46 @@
+'''
+Plugins
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Web Application Scanning v3 plugins <was-v2-plugins>` API.
+
+Methods available on ``tio.v3.was.plugins``:
+
+.. rst-class:: hide-signature
+.. autoclass:: PluginsAPI
+    :members:
+'''
+from typing import Dict
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class PluginsAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/was/plugins'
+    _conv_json = True
+
+    def plugin_details(self, id: int) -> Dict:
```

Comment:
We don't have other methods here. I think we should keep the name as details instead of plugin_details.

Reply:
Updated